jpo-asn-j2735-2024/src/main/java/us/dot/its/jpo/asn/j2735/r2024/TravelerInformation/ITIStextPhrase.java

```diff
@@ -34,6 +34,6 @@ public ITIStextPhrase() {
   @JsonCreator
   public ITIStextPhrase(String value) {
     this();
-    this.value = value;
+    this.setValue(value);

```

Comment:
```suggestion
    this.value = value;
```
issue (blocking): ITIStextPhrase is an IA5String, not an Asn1Integer; it is out of scope to change